### PR TITLE
Fix many ActiveRecord deprecation warnings

### PR DESCRIPTION
**Pull request by ur5us**
This does not fix all of them but the ones that seem reasonably safe without
being able to run the specs yet.

### CODE REVIEW

**Review by alagos — APPROVED** [review]
Oh, I made almost these same changes for the last meetup, but I never pushed them :sweat_smile:

File: app/models/group.rb [hunk 1]
```diff
@@ -36,20 +36,23 @@ class Group < ActiveRecord::Base
...
   has_one :forum
   has_one :privacy_setting
-  has_many :reqs, :conditions => ["biddable = ?",true], :order => "created_at DESC"
-  has_many :offers, :order => "created_at DESC"
-  has_many :photos, :as => :photoable, :dependent => :destroy, :order => "created_at"
-  has_many :exchanges, -> { order("created_at DESC") }
-  has_many :exchange_and_fees, :order => "created_at DESC"
+  has_many :reqs, -> { where(biddable: true).order(created_at: :desc) }
+  has_many :offers, -> { order(created_at: :desc) }
+  has_many :photos, -> { order(:created_at) }, :as => :photoable, :dependent => :destroy
```
Comment (lesliev) [inline app/models/group.rb:41]: I suppose you can use the old hashrocket syntax if you MUST :)

File: app/models/group.rb [hunk 2]
```diff
@@ -36,20 +36,23 @@ class Group < ActiveRecord::Base
...
-  has_many :people, :through => :memberships, 
-    :conditions => "status = 0", :order => "name DESC"
-  has_many :pending_request, :through => :memberships, :source => "person",
-    :conditions => "status = 2", :order => "name DESC"
-  
+  has_many :people,
+           -> { where(status: 0).order(name: :desc) },
+           :through => :memberships
+  has_many :pending_request,
+           -> { where(status: 2).order(name: :desc) },
```
Comment (lesliev) [inline app/models/group.rb:50]: Weird that this 2 is not in a constant but let's leave it for now.

**Comment by lesliev** [comment]
Awesome, thanks a lot!

File: app/models/person.rb [hunk 3]
```diff
@@ -175,32 +175,48 @@ def subscribed_to_stripe
...
   has_many :feeds
-  has_many :activities, :through => :feeds, :order => 'activities.created_at DESC',
-  :limit => FEED_SIZE,
-  :conditions => ["people.deactivated = ?", false],
-  :include => :person
-
-  has_many :own_groups, :class_name => "Group", :foreign_key => "person_id", :order => "name ASC"
+  has_many :activities,
+           -> {
+            inlcude(:person)
```
Comment (alagos) [inline app/models/person.rb:201]: Just in case, I fixed this typo in a becoming PR

File: app/models/group.rb (hunk 1 above)
Comment (ur5us) [reply app/models/group.rb:41]: Yeah, I didn't want to change too many things at once. Changing hash rocket syntax to JSON syntax can be easily automated with RuboCop.

**Pull request merged**